rules/falco_rules.yaml

```diff
@@ -2604,7 +2604,25 @@
   output: "Docker or kubernetes client executed in container (user=%user.name %container.info parent=%proc.pname cmdline=%proc.cmdline)"
   priority: WARNING
   tags: [container, mitre_execution]
+
+
+# This rule is not enabled by default, as there are legitimate use
+# cases for raw packet. If you want to enable it, modify the
+# following macro.
+- macro: consider_raw_packet_communication
+  condition: (never_true)
+
+- list: user_known_raw_packet_binaries
+  items: []
+
+- rule: Raw packet created in container
```

Comment:
Let's rename to `Packet socket created in container` :)

Reply:
Thank you :)
I renamed them including macro, list name and desc of the rule